components/bl-drawer-component/src/index.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useRef } from 'react';
+
+import { useCloseEvents, useDrawerActions, useDrawerStyles } from './helpers';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function DrawerComponent({ component, eventHandlers, pods, elRef }) {
+  const { classList, display, style, openDrawerButtonVisibility, openDrawerButtonLabel } = component;
+
+  const drawerContentRef = useRef(null);
+  const drawerContainerRef = useRef(null);
+
+  const {
+    visibility, openContent, closeContent,
+  } = useDrawerActions(component, eventHandlers, drawerContentRef, drawerContainerRef);
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div ref={ elRef } className={ cn('bl-customComponent-drawer', classList) } style={ style }>
+      { openDrawerButtonVisibility && (
+        <button type="button" onClick={ openContent }>{ openDrawerButtonLabel }</button>
```

Comment:
will be good to add `aria-label` that describe this button functionality

'open backdrop menu' or something like that